rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -89,15 +90,25 @@
             sw360Releases.addAll(releaseService.getReleasesForUser(sw360User));
         }
 
+        sw360Releases = sw360Releases.stream()
+                .filter(release -> name == null || name.isEmpty() || release.getName().equals(name))
+                .collect(Collectors.toList());
+
         List<Resource> releaseResources = new ArrayList<>();
         for (Release sw360Release : sw360Releases) {
             Release embeddedRelease = restControllerHelper.convertToEmbeddedRelease(sw360Release, fields);
             Resource<Release> releaseResource = new Resource<>(embeddedRelease);
             releaseResources.add(releaseResource);
         }
-        Resources<Resource> resources = new Resources<>(releaseResources);
 
-        return new ResponseEntity<>(resources, HttpStatus.OK);
+        Resources<Resource> resources = null;
```

Comment:
Is it possible to make it generic for all the resource like projects, components etc by moving it to `RestControllerHelper.java` and also it would be good to make the response status consistence for all other resource which is returning  response without content.

Reply:
Changes made.